Resources/Prototypes/Entities/Mobs/Species/reptilian.yml

```diff
@@ -0,0 +1,218 @@
+- type: entity
+  save: false
+  name: Urist McHands The Lizardperson
+  parent: MobHumanBase
+  id: MobReptilianBase
+  abstract: true
+  description: A miserable pile of scales.
+  components:
```

Comment:
I merged in master but the reptilian mob still had doorbumpopener component which tells me there needs to be more shared with humans.

Reply:
Same with innate actions being duplicated across reptile.

Reply:
As is, we can't automatically reuse/merge the sprite layers due to how prototypes work, so I don't think anything more CAN be shared.

Reply:
Doesn't the entire sprite get bulldozed? That seems okay to me.

Master shouldn't have to care about manually ensuring that reptiles and humans are copy-pasted exactly.

Reply:
> Doesn't the entire sprite get bulldozed? That seems okay to me.
> 
> Master shouldn't have to care about manually ensuring that reptiles and humans are copy-pasted exactly.

I'll note we already have this issue with the other species, too. It's something that'll be solved when composition is in.